### DISCUSSION TITLE
How can I save all file names in a playlist in the watch later file?

### DISCUSSION THREAD

**Discussion by ruihe774** [mpv-player/mpv#14142] [Q&A]
I usually play multiple episodes with mpv. I use the following command:

```
mpv *.mkv
```

I often delete watched episodes afterwards. However, I sometimes forget which episodes I have watched when I want to download new episodes.

I have tried `--write-filename-in-watch-later-config`. But it only saves one current playing filename in the watch later file.

I'm wondering if it is possible to save all file names in a playlist in the watch later file?

**Comment by lazypingu** [comment]
Not a direct solution but maybe a playlist file (e.g. `episodes.m3u`) which contains all files could be used instead?
